spec/query_dsl_spec.rb

```diff
@@ -0,0 +1,17 @@
+RSpec.describe "QueryDSL" do
+  it "converts params to ES DSL" do
+    dsl = QueryDSL.new(params: {q: 'foo', aq: { color: "green" } }, current_user: nil)
+    expect(dsl.to_hash).to eq({
+      query: {
+        query_string: {
+          query: "(foo) AND (color:(green))",
+          default_operator: "and"
+        },
+      },
+      size: QueryDSL::MAX_RESULTS,
+      from: 0
+    })
+  end
+
+  # TODO pass with current_user and test filter
```

Comment:
this card sums it up: https://trello.com/c/0RRTt2mN

right now there is no sense of a `current_user` in the app, but when there is, we'll want to be able to add a ES filter to limit search results by authorization.